docs/components/tpr-header-bar.md

```diff
@@ -60,6 +61,21 @@ Typically used for a menu of relevant links.
 
 Must be inside a `<govuk-header-bar>` element.
 
+
+### <tpr-header-search>
+|    Attribute      | Type   | Description                                                           |
+|-------------------| ------ | ----------------------------------------------------------------------|
+|     `action`      |`string`| Sets action path of search form submission.                           |
+|`autocomplete-url` |`string`| Sets endpoint for autocomplete.js fetch operation.                    |
+|   `placeholder`   |`string`| Supports setting custom placeholder for generated input elements.     |
+|   `aria-label`    |`string`| Programmatically setting aria-label value of input and button elements|
+
+TPR Header Search will display when there is no header content and DisplayHeaderSearch property is set to true. On smaller screen sizes the tpr mobile menu component will take over, and the header search will display as part of the mobile menu when expanded.
```

Comment:
TPR should be in caps, and you should also explain that the mobile behaviour is not yet implemented and therefore this is currently a desktop-only component.